### DISCUSSION TITLE
How do I pause a project without waiting for a week?

### DISCUSSION THREAD

**Discussion by WaterlessPiano2** [supabase/supabase#9152] [Questions] [answered]
Today I want to create a new project, but Supabase is asking me to delete or pause one of my other 2 projects. I don't want to permanently delete any of them, but I am happy to pause one.

Is there a way to pause a project imminently, on demand? if yes, how do I do this?

**Comment by GaryAustin1** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/54564956/192147878-4d014a09-ea16-4858-9290-9edbac0f5368.png)

**Reply by aaronksaunders** [reply]
this UI has changed, there is no longer a pause option available

**Reply by GaryAustin1** [reply]
@aaronksaunders Maybe you are not on a free plan?  Pause is only for free plans.
![image](https://user-images.githubusercontent.com/54564956/213895487-1540fad5-dd3c-42ed-b0cc-6ddf88d94eb6.png)

**Comment by aaronksaunders** [comment]
I am not on the free plan but I want to pause a running project to start
one of my test projects since it won’t let me spin up the other project?

On Sat, Jan 21, 2023 at 7:41 PM GaryAustin1 ***@***.***>
wrote:

> @aaronksaunders <https://github.com/aaronksaunders> Maybe you are not on
> a free plan? Pause is only for free plans.
> [image: image]
> <https://user-images.githubusercontent.com/54564956/213895487-1540fad5-dd3c-42ed-b0cc-6ddf88d94eb6.png>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/supabase/supabase/discussions/9152#discussioncomment-4747408>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAEAFGIIYVWWKHJ4YUID5TLWTR63XANCNFSM6AAAAAAQVDDGKE>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
-- 

--

Aaron K. Saunders
CEO Clearly Innovative Inc
***@***.***
www.clearlyinnovative.com


This email message and any attachment(s) are for the sole use of the
intended recipient(s) and may contain proprietary and/or confidential
information which may be privileged or otherwise protected from disclosure.
Any unauthorized review, use, disclosure or distribution is prohibited. If
you are not the intended recipient(s), please contact the sender by reply
email and destroy the original message and any copies of the message as
well as any attachment(s) to the original message.

**Reply by GaryAustin1** [reply]
Then you need to contact support and see if they can help.  I've not seen Supabase's reasoning, but my guess is because pause/restore will bring in latest versions of API servers and Postgres and that could break production systems.

**Comment by tananga** [comment]
@GaryAustin1 Can I get an information if project is paused from API?  I would like to show a message on my website if project is paused

**Reply by GaryAustin1** [reply]
I'm not aware of a way to know it is paused.
The Project management API at a quick glance does not show anything useful.

You could certainly see if the database or auth is responding by doing a health check like the dashboard does by polling these endpoints.
![image](https://github.com/supabase/supabase/assets/54564956/68afa4f1-6cb4-4ed6-8adb-210773d2df58)

**Comment by remorses** [comment]
Can you enable the pause button for Supabase projects on a subscription? I have many projects for staging environment and don't want to keep them always running
